CMakeLists.txt

```diff
@@ -49,8 +49,8 @@ if(${ament_cmake_FOUND})
   )
 
   install(
-    DIRECTORY scripts/
-    DESTINATION bin
+    PROGRAMS scripts/rttest_plot
+    DESTINATION bin/rttest_plot
```

Comment:
Have you tried this locally? I am pretty sure it will install the script into a subfolder under bin.